### ISSUE TITLE
Post to database table running in docker

### ISSUE THREAD

**Issue by IndraPhani** [Da9el00/Spring-Boot-JPA-Docker-MySQL#1]
1. I have tried the post method and expecting a response with the URI and HTTPStatus code, below is my method signature.
2. `@PostMapping("/adding")  
      public ResponseEntity handelPostIntoDatabase(UserDTO uDto){
//    	  UserEntity userEntity = userMapper.UserDTOToUserEntity(uDto);// Mapper class to covert DTO object to Entity object
//        userRepository.save(userEntity);
//       // HttpHeaders headers = new HttpHeaders();
//       // headers.add("Location", usersavedDto.toString());
//        return new ResponseEntity<>(HttpStatus.CREATED);
//    }`

I have an exception saying the Column name cannot be empty, but i was sending the complete object details for DTO object from postman as json show below.
{
    "userID":"4"
    "name":"Hi",
    "email":"hi@gmail.com"
}

In a nutshell I want to check if I send the data which is in json format to application will it store in the script or any where or not.

**Comment by IndraPhani** [comment]
Changed the entity basic setup and uploaded the empty, .sql file to intialize once the docker MySql container started, solved the issue.
